setup.py

```diff
@@ -13,7 +13,7 @@
 
 setup(
     name='ledcontroller',
-    version='1.3.0',
```

Comment:
Please don't include version bumps to your PRs - I'll decide on the version numbers and release times separately.